Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -926,28 +926,32 @@ - (CGSize)bodyMediaSizeForContentWidth:(int)contentWidth
             }
 
             return CGSizeRound(CGSizeMake(mediaWidth, mediaHeight));
+            break;
         }
         case OWSMessageCellType_Audio:
-            return CGSizeMake(maxMessageWidth, OWSAudioMessageView.bubbleHeight);
+            result = CGSizeMake(maxMessageWidth, OWSAudioMessageView.bubbleHeight);
+            break;
         case OWSMessageCellType_GenericAttachment:
-            return CGSizeMake(maxMessageWidth, [OWSGenericAttachmentView bubbleHeight]);
+            result = CGSizeMake(maxMessageWidth, [OWSGenericAttachmentView bubbleHeight]);
+            break;
         case OWSMessageCellType_DownloadingAttachment:
-            return CGSizeMake(200, 90);
+            result = CGSizeMake(200, 90);
+            break;
         case OWSMessageCellType_ContactShare:
             OWSAssert(self.viewItem.contactShare);
 
-            return CGSizeMake(
+            result = CGSizeMake(
                 maxMessageWidth, [OWSContactShareView bubbleHeightForContactShare:self.viewItem.contactShare]);
+            break;
     }
-}
 
-- (int)maxMessageWidthForContentWidth:(int)contentWidth
-{
-    return (int)floor(contentWidth * 0.8f);
+    return CGSizeCeil(result);
```

Comment:
I'm trying to move towards using `CGSizeCeil()` to ensure point-align our layouts where possible.